README.md

```diff
@@ -1,13 +1,27 @@
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+<a href="https://comit.network">
```

Comment:
Note: this link is to be updated to point to the SDK documentation section once it exists.